CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 - Fix transport in autoprovision device depending on binding [#235]
-- Allow get list of commands without sending measures (empty payload) [#234]
\ No newline at end of file
+- Allow get list of commands without sending measures (empty payload) [#234]
+- Add: allow NGSIv2 for updating active attributes at CB, throught configuration based on iotagent-node-lib(#250)
```

Comment:
Issue number seems to be wrong, I think it is #233 (seems to be a copy-paste error, as 250 is the corresponding one in the iota-json repository).

Reply:
Not really... commit 93cfad414288b9f918557d239905100dd5d6976c doesn't touch CHANGES_NEXT_RELEASE file